### DISCUSSION TITLE
Why convert a .jpg photo to .avif(lossless), the output file is bigger than the original one

### DISCUSSION THREAD

**Discussion by cruvie** [libvips/libvips#3464] [Q&A] [answered]
1. Use Image [IMG_20230425_210513.jpg.zip](https://github.com/libvips/libvips/files/11322896/IMG_20230425_210513.jpg.zip)
2. Use Configuration 
```go
ep := vips.NewAvifExportParams()
ep.Quality = 100
ep.Lossless = true
ep.Speed = 9 
ep.StripMetadata = false
```

**Environment**
 - OS:  MacOS 13.1 M2
 - Vips: 8.12.0

**Comment by jcupitt** [comment, marked as answer]
Hi @gkdgo,

Lossless AVIF is more like PNG than JPEG. Try:

```
$ vips copy IMG_20230425_210513.jpg x.png
$ vips copy IMG_20230425_210513.jpg x.avif[lossless]
$ ls -l x.png x.avif
-rw-r--r-- 1 john john  7749739 Apr 25 15:08 x.avif
-rw-r--r-- 1 john john 17648791 Apr 25 15:07 x.png
```

Your JPEG image is about Q 95 with chroma subsample, so the equivalent AVIF might be:

```
$ vips copy IMG_20230425_210513.jpg x.avif[Q=80,subsample_mode=on]
$ ls -l IMG_20230425_210513.jpg x.avif
-rw------- 1 john john 4574770 Apr 25 14:05 IMG_20230425_210513.jpg
-rw-r--r-- 1 john john 1806023 Apr 25 15:10 x.avif
```

**Reply by cruvie** [reply]
Thank u @jcupitt , I  understand, could you please tell how get jpg file's Q? I searched the web and use photoshop ,but I get nothing, and why the equivalent AVIF Q is 80? Is there a Conversion formula or just by viewing output file?

**Reply by jcupitt** [reply]
I tried recompressing your file, and Q 95 is about the same:

```
$ vips copy IMG_20230425_210513.jpg x.jpg[Q=95,subsample_mode=on]
$ ls -l IMG_20230425_210513.jpg x.jpg 
-rw------- 1 john john 4574770 Apr 25 14:05 IMG_20230425_210513.jpg
-rw-r--r-- 1 john john 4583618 Apr 25 15:06 x.jpg
```

AVIF Qs are quite a bit smaller than JPEG ones, so I was just guessing.

**Reply by cruvie** [reply]
ok, is it possible to evaluate jpg's Q in vips? I just want to convert jpg to avif with as possible as the same quality, but now it seems I can just do it manually.

**Reply by jcupitt** [reply]
JPEGs don't have the Q recorded -- you can only estimate it. Imagemagick tries to estimate Q from an analysis of the coding tables, but it can be very inaccurate.

I think I would just use the default settings.

**Reply by cruvie** [reply]
got it thanks!
